student.rb

```diff
@@ -1,14 +1,15 @@
-require_relative './person'
+require_relative 'person'
+require_relative 'classroom'
 
 class Student < Person
-  attr_accessor :classroom
+  attr_reader :classroom
 
   def initialize(_classroom, age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
   end
 
-  def owner=(classroom)
+  def classroom=(classroom)
     @classroom = classroom
     classroom.students.push(self) unless classroom.students.include?(self)
   end
```

Comment:
- [ ] I don't know why you do have a definition for the classroom. but I think it's not necessary.